astropy/coordinates/transformations/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Coordinate transformations."""
+
+from astropy.coordinates.transformations.affine import (
+    AffineTransform,
+    BaseAffineTransform,
+    DynamicMatrixTransform,
+    StaticMatrixTransform,
+)
+from astropy.coordinates.transformations.base import CoordinateTransform
+from astropy.coordinates.transformations.composite import CompositeTransform
+from astropy.coordinates.transformations.function import (
+    FunctionTransform,
+    FunctionTransformWithFiniteDifference,
+)
+from astropy.coordinates.transformations.graph import TransformGraph
+
+# Backward compatibility imports
+# isort: split
+from astropy.coordinates.transformations.graph import trans_to_color
```

Comment:
Backward compatibility with what? This was never publicly exposed, so I'd be happy to just not have it here at all.

Reply:
✅ 
Happy to remove.